assemble/bin/accumulo-cluster

```diff
@@ -80,6 +80,11 @@ function parse_args {
   ARG_COMPACTOR_GROUP=""
 
   PARSE_OUTPUT=$(getopt -o "" --long "dry-run,all,local,manager,gc,monitor,compaction-coordinator,no-tservers,tservers,sservers::,compactors::" -n 'accumulo-cluster' -- "$@")
+  if [[ "$?" != 0 ]]; then
+    print_usage
+    exit 1
+  fi
+
```

Comment:
So this will just error whenever getopt sees an unrecognized option, then? Like, if you put `-o someoption=somevalue` here, incorrectly expecting it to be added to the service command line?

Reply:
Yes. The current behavior of getopt is not modified as it currently produces an "Unrecognized Option: <x>"  message and remove it from future processing. 

This change just means the script will now exit instead of proceeding with a, possibly unintended, cluster command.